### PR TITLE
Update S2WidthCut for SR1

### PR DESCRIPTION
**Pull request by pdeperio**
New PR targeting master branch now instead of https://github.com/XENON1T/lax/pull/59

### CODE REVIEW

**Review by feigaodm — COMMENTED** [review]
With drift field changed, the drift velocity and diffusion constant is different. My question is: Is this cut class using the correct value of those quantities?

**Comment by pdeperio** [comment]
@weiyuehuan is working on updating to the correct numbers still.

**Comment by weiyuehuan** [comment]
@feigaodm @pdeperio  I think the last pull is correct ?  This definition is based on the drift velocity and diffusion constant in xenon1t.ini file, which is old value.   Just use it like this is fine.   I have calculated the new drift velocity and diffusion constant. If we update the new values, I will also change the this cut definition.  The new drift velocity and diffusion constant can be found:
https://xecluster.lngs.infn.it/dokuwiki/doku.php?id=xenon:xenon1t:yuehuan:analysis:1sciencerun_s2width#appendix_2drift_velocity_and_diffusion_constant

**Comment by pdeperio** [comment]
@weiyuehuan sorry for the spread out discussions, but in the other thread (https://github.com/XENON1T/lax/issues/61), we decided to hardcode the new physics values within ```sciencerun1.py```, like:
```
diffusion_constant = 31.73  * units.cm**2 / units.s  # ± 0.05
v_drift = 1.3351 * units.mm / units.us # ± 0.0023
```
instead of:
```
diffusion_constant = PAX_CONFIG['WaveformSimulator']['diffusion_constant_liquid']
v_drift = PAX_CONFIG['DEFAULT']['drift_velocity_liquid']
```
(Then I guess you should also remove the scale factor in the function that follows.)

Is this OK?

**Comment by weiyuehuan** [comment]
@pdeperio sure, I can remove the scale factor, just tell me the time I can do it.  

I also need to check the scale factor again,  remember Fig 3 in this link:
https://xecluster.lngs.infn.it/dokuwiki/doku.php?id=xenon:xenon1t:yuehuan:analysis:0sciencerun_s2width_update0#comparison_with_diffusion_model_cut_by_jelle_pax_v642
Jelle calculate the drift velocity and diffusion constant for SR0, but I still need to scale the parameter to other value.

**Comment by pdeperio** [comment]
You can modify this (your) branch any time until we decide to merge it here.

Can you remind me where [this scale factor](https://github.com/XENON1T/lax/blob/v0.11.1/lax/lichens/sciencerun0.py#L584) (```4.0325 ```) comes from?   According to @JelleAalbers' [note](https://xecluster.lngs.infn.it/dokuwiki/doku.php?id=xenon:xenon1t:aalbers:drift_and_diffusion#diffusion_constant), I thought this is just a constant scaling from Gaussian sigma to our width parameter ```r50```.  But according to [your note](https://xecluster.lngs.infn.it/dokuwiki/doku.php?id=xenon:xenon1t:yuehuan:analysis:1sciencerun_s2width), I guess you're also allowing it to float in the fit. 
 (Also, I don't see ```w0 = 348.6 * units.ns``` in Jelle's note or notebook therein, but I guess is also fitted.)

**Comment by JelleAalbers** [comment]
In my note w0 was only shown in the corner plot linked under error estimation (direct link here: https://xecluster.lngs.infn.it/dokuwiki/lib/exe/detail.php?id=xenon%3Axenon1t%3Aaalbers%3Afirst_single_electrons&media=xenon:xenon1t:aalbers:drift_and_diffusion:D_corner_3068-3086_.png )

Yuehuan told me that since he couldn't modify the diffusion constant and drift velocity in pax's ini, he increased the theoretical prefactor 3.6395 from https://xecluster.lngs.infn.it/dokuwiki/doku.php?id=xenon:xenon1t:aalbers:drift_and_diffusion#diffusion_constant. From the ratio of Yuehuan's number and this number, and the ratios between the old and new diffusion constants and drift velocities, we can derive the new diffusion constant.

**Comment by weiyuehuan** [comment]
Yes, In Jelle's note, the  value are 3.6395 (scaling from Gaussian sigma to our width parameter r50) and w0 = 304.  
When I defined SR0 cut, I found these parameters can't fit our data, (https://xecluster.lngs.infn.it/dokuwiki/doku.php?id=xenon:xenon1t:yuehuan:analysis:0sciencerun_s2width_update0#comparison_with_diffusion_model_cut_by_jelle_pax_v642). so I adjust these values to 4.0325 and 348.6.

In this pull, I adjust these values again to let it fit SR1 data. the performance can be found in this plot:
cut_definition

If We keep Jelle's value, 3.6395 and 304, and using the new Diffusion constant and drift velocity that I calculated, the cut will be as below, which similar as Fig 3 in my  old note. The centre still not sat at y = 1.
new_diffusion_constant_and_drift_velocity

So  I think  if we want to update the diffusion constant and drift velocity, I will update the scaler factor accordingly.

File: lax/lichens/sciencerun1.py [hunk 1]
```diff
@@ -177,7 +175,64 @@ def pre(self, df):
...
+        self.lichen_list = [self.S2WidthHigh(),
+                            self.S2WidthLow()]
+
+    @staticmethod
+    def s2_width_model(z):
+        diffusion_constant =  31.73 * ((units.cm)**2) / units.s 
+        v_drift = 1.335 * (units.um) / units.ns
+        GausSigmaToR50 = 1.349
+
+        EffectivePar = 1.16
```
Comment (pdeperio) [inline lax/lichens/sciencerun1.py]: More precision?

**Comment by feigaodm** [comment]
Is this cut ready to be merged? Need it for analysis like accidental coincidence background estimation.

**Comment by weiyuehuan** [comment]
the cut is fine, but lax format has problem. I tried n times, but still can't fix it....

**Review by feigaodm — APPROVED** [review]

**Comment by feigaodm** [comment]
yeah, looks like the style issue has been fixed by @pdeperio, will merge it shortly. Thanks

**Pull request merged**